deepaas/cmd/execute.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import shutil
+import ntpath
+import os
+import collections
+
+from deepaas.model import loading
+from oslo_log import log
+
+LOG = log.getLogger(__name__)
+
+### Loading the model installed 
+def model_name():
+    global MODEL_NAME
+    try:
+        for name, model in loading.get_available_models("v2").items():
+            MODEL_NAME = name
+    except Exception as e:
+        LOG.warning("Error loading models: %s",e)
+
+
+
+def main():
+    model_name() #Function to know the name of the installed model
+
+    package_name = MODEL_NAME
+    predict_data=__import__(package_name).api.predict_data  #import function
+    predict_url=__import__(package_name).api.predict_url  #import function
+
+    parser = argparse.ArgumentParser (
```

Comment:
We are using oslo.cfg for handling options, command line options and positional arguments. Converting from argparse to oslo.cfg is quite easy, and that way we can retain other options defined in other parts of the code.

For an example, see https://github.com/indigo-dc/DEEPaaS/blob/master/deepaas/cmd/wsk.py#L28